x-pack/legacy/plugins/siem/public/pages/case/components/user_action_tree/user_action_title.tsx

```diff
@@ -37,32 +43,106 @@ export const UserActionTitle = ({
   isLoading,
   labelAction,
   labelTitle,
+  linkId,
   userName,
+  updatedAt,
   onEdit,
+  outlineComment,
 }: UserActionTitleProps) => {
+  const { detailName: caseId } = useParams();
+  const urlSearch = useGetUrlSearch(navTabs.case);
   const propertyActions = useMemo(() => {
-    return [
-      {
-        iconType: 'documentEdit',
-        label: labelAction,
-        onClick: () => onEdit(id),
-      },
-    ];
-  }, [id, onEdit]);
+    if (labelAction != null && onEdit != null) {
+      return [
+        {
+          iconType: 'documentEdit',
+          label: labelAction,
+          onClick: () => onEdit(id),
+        },
+      ];
+    }
+    return [];
+  }, [id, labelAction, onEdit]);
+
+  const handleAnchorLink = useCallback(() => {
+    copy(`${window.location.origin}${window.location.pathname}#case/${caseId}/${id}${urlSearch}`, {
```

Comment:
nit: We can retrieve `#case` from `x-pack/legacy/plugins/siem/public/pages/home/types.ts`.